transformers/src/unit-test/java/uk/gov/moj/sdt/transformers/UpdateItemTransformerTest.java

```diff
@@ -68,6 +71,8 @@ public class UpdateItemTransformerTest extends AbstractSdtUnitTestBase {
     /**
      * Set up variables for the test.
      */
+    @BeforeEach

```

Comment:
Is the BeforeEach annotation needed?  The setUpLocalTests() method is invoked from the setUp() method on the parent AbstractSdtUnitTestBase class which itself is annotated with BeforeEach.  This looks like it should result in setupLocalTests() being called before each test.

Reply:
Done